packages/lodestar/src/network/network.ts

```diff
@@ -59,7 +59,8 @@ export class Libp2pNetwork extends (EventEmitter as { new(): NetworkEventEmitter
         this.peerInfo = libp2p.peerInfo;
         this.libp2p = libp2p;
         this.reqResp = new ReqResp(opts, {config, libp2p, logger});
-        const enr = (this.libp2p._discovery.get("discv5") as Discv5Discovery)?.discv5?.enr || undefined;
```

Comment:
curious about this `?.` syntax, is it for newer node version?

Reply:
optional chaining, not supported by nodejs until october 2020